firecrown/ccl/two_point.py

```diff
@@ -91,7 +91,7 @@ def compute_loglike(
     else:
         loglike = None
 
-    return loglike, None
+    return loglike, {'data': _data, 'theory': _theory, 'loglike': loglike}
```

Comment:
This change means there needs to be doc change in the function below.

Reply:
done.